imqsauth.go

```diff
@@ -234,6 +251,221 @@ func createDB(config *authaus.Config) bool {
 	return true
 }
 
+func healthCheck(ic *auth.ImqsCentral, cleanAll bool) bool {
```

Comment:
Writing this comment here instead of on the overall review, as refreshing my page seems to throw away that comment :(

This repo unfortunately suffers from not having a lot of packages, and so knowing where to put new code is a bit difficult. I want you to move this function, and its friends, that it is depended on, to a new package called `health`. Some of the functions might fit better into a package called `user`, but there might be code in the `auth` package that should also be there, and then it becomes hard to understand the repo with code being not where you would expect it.

So for now I think we just move it to a new package called `health`, this would also have the positive affect of moving the new struct, and more importantly, the new constant variables that you have made to a different file, as they feel a bit weird in this file.

All of this being said, I do realize that the pattern for this repo is clearly to place code used for the CLI in this file to keep things simple, but I think what you have written here is a lot more complex than any of the other CLI functions, so it feels right to put it somewhere else.

Reply:
Makes sense.